### PR TITLE
Add ciphercmd a command line tool to use cipher

### PR DESCRIPTION
**Pull request by fortizc**
**WIP:** this PR isn't finished, please not merge

**Whats is this?**
This is the cipher command-line too, right now allows encrypt and decrypt files using a key file

**Missing features**
Some features needed to make this PR awesome:
* Create a way to install the CMD and create it an entry point.
* Receive string from the command line directly, e.g: `echo "encrypt me" | ciphercmd -e `

@pahumadad can you help me with any of the missing features?

### CODE REVIEW

File: tool/ciphercmd.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    txt = cipher.read(fname, key)
+
+    with open(out, "w") as fd:
+        fd.write(txt)
+
+
+def main():
+    parser = parse_args()
+    args = parser.parse_args()
+
```
Comment (fortizc) [inline tool/ciphercmd.py]: I don't know if there is a better way to implement this

**Comment by pahumadad** [comment]
of course, I can help! ;)

**Comment by pahumadad** [comment]
now, for example, you can do:
```
$ echo 'Hello, World!!!' | ciphercmd encrypt -s asdf -i "$(cat -)"
WDJoMWJYVXhkWE5mWlc1algxOWkzSGN4Z3luekZybEQvK1djNEUvQ1h0U3RId1Jjb2ZFPQ==
$ echo 'Hello, World!!!' | ciphercmd encrypt -s asdf -i "$(cat -)" > out.secret
$ cat out.secret | xargs ciphercmd decrypt -s asdf -i
Hello, World!!!
$ cat out.secret | ciphercmd decrypt -s asdf -i $(cat -)
Hello, World!!!
```

**Comment by pahumadad** [comment]
Now, you can do:
```
$ echo 'Hello, World!!!' | ciphercmd encrypt -i
WDJoMWJYVXhkWE5mWlc1algxOHJPaEZObnBJRjBXYUdxTlJhKzdOMW5ycUpCcUNaL3BJPQ==

$ echo 'Hello, World!!!' | ciphercmd encrypt -i | ciphercmd decrypt
Hello, World!!!

$ echo 'Hello, World!!!' | ciphercmd encrypt -i > file.encrypted
```

**Review by fortizc — COMMENTED** [review]
:+1:

**Review by pahumadad — APPROVED** [review]
👍

**Pull request merged**